server/src/main/java/io/spine/server/aggregate/AggregateEndpoint.java

```diff
@@ -177,7 +180,7 @@ private DispatchOutcome applyProducedEvents(A aggregate, DispatchOutcome command
             event.getContextBuilder()
                  .setVersion(signalVersion);
         }
-        return correctedCommandOutcome.vBuild();
+        return correctedCommandOutcome.build();
```

Comment:
Please use `buildPartial` here and in similar cases. We should explicitly state the absence of validation.

Reply:
I view this differently. Let's discuss in person.

Reply:
The absence of `v` prefix does this.